doc/chapters/integration-guidelines/index.rst

```diff
@@ -19,8 +19,9 @@ Service manifests
 For details about format and content of service manifests, see :ref:`Service manifests <service-manifests>`
 and :ref:`Gateways <gateways>`.
 
-Service manifests should be installed in ``/etc/softwarecontainer/caps.d/<service-name>`` and are
-expected to be of type JSON, including the "json" file extension.
+Service manifests should be installed in either ``/etc/softwarecontainer/caps.d/<service-name>`` or
+``/etc/softwarecontainer/caps.default.d/<service-name>`` depending on if it should be applied by default
+or not. Each manifest are expected to be of type JSON, including the "json" file extension.
```

Comment:
Shouldn't this be ${FULL_SYSCONFDIR}/softwarecontainer/...? We can do substitutions from cmake to the sphinx config file for things like this, please do.